addons/l10n_ru/data/account_account_template_data.xml

```diff
@@ -0,0 +1,5979 @@
+      "Номенклатура".
+    </field>
+  </record>
+  <record id="ru_acc_004" model="account.account.template">
+    <field name="name">Товары, принятые на комиссию</field>
+    <field name="code">004</field>
+    <field name="reconcile" eval="False" />
+    <field name="chart_template_id" ref="chart_template" />
+    <field name="user_type_id" ref="account.data_account_off_sheet" />
+    <field name="note">
+      Счет 004 "Товары, принятые на комиссию" предназначен для обобщения
+      информации о наличии и движении товаров, принятых на комиссию в
+      соответствии с договором. Этот счет используется
+      организациями-комиссионерами. Аналитический учет по счету ведется по
+      партиям (субконто "Партии") и видам товаров (субконто "Номенклатура").
+      Каждый вид товара - элемент справочника "Номенклатура".
+    </field>
+  </record>
+  <record id="ru_acc_004_01" model="account.account.template">
+    <field name="name">Товары на складе</field>
+    <field name="code">004.01</field>
+    <field name="reconcile" eval="False" />
+    <field name="chart_template_id" ref="chart_template" />
+    <field name="user_type_id" ref="account.data_account_off_sheet" />
+    <field name="note">
+      Субсчет 004.01 "Товары на складе" предназначен для обобщения информации о
+      наличии и движении товаров, принятых на комиссию в соответствии с
+      договором. Аналитический учет ведется по партиям (субконто "Партии"),
+      видам товаров (субконто "Номенклатура") и местам их хранения (субконто
+      "Склады"). Каждый вид товара - элемент справочника "Номенклатура". Каждое
+      место хранения - элемент справочника "Склады (места хранения)". Для
+      ведения аналитического учета по субконто "Склады" следует выполнить
+      соответствующие настройки параметров учета (меню "Предприятие" -
+      "Настройка параметров учета" - "Запасы").</field
+    >
+  </record>
+  <record id="ru_acc_004_02" model="account.account.template">
+    <field name="name">Товары, переданные на комиссию</field>
+    <field name="code">004.02</field>
+    <field name="reconcile" eval="False" />
+    <field name="chart_template_id" ref="chart_template" />
+    <field name="user_type_id" ref="account.data_account_off_sheet" />
+    <field name="note">
+      Субсчет 004.02 "Товары, переданные на комиссию" предназначен для обобщения
+      информации о наличии и движении товаров, принятых на комиссию и переданных
+      для реализации комиссионеру. Аналитический учет ведется по партиям
+      (субконто "Партии"), комиссионерам (субконто "Контрагенты") и видам
+      товаров (субконто "Номенклатура"). Каждый вид товара - элемент справочника
+      "Номенклатура". Каждый комиссионер - элемент справочника
+      "Контрагенты".</field
+    >
+  </record>
+  <record id="ru_acc_004_K" model="account.account.template">
+    <field name="name"
+      >Корректировка комиссионных товаров прошлого периода</field
+    >
+    <field name="code">004.K</field>
```

Comment:
I'm not sure about the truth
But in 1C account "004.K" named with Russian letter "К". And there are several accounts with similar problem like "76.AB"
For me it's important because I'm writing sync module between 1C and Odoo.

Reply:
Hello!
Me and Ivan were discussing about recent commit, that denies to use cyrillic characters in "code" field. If you speak Russian, here is discussion itself: https://github.com/odoo/odoo/commit/623d3bfe0fa5db936a05cd17e484422efea86464#r82761697